### PR TITLE
accessibility updates

### PR DESCRIPTION
**Pull request by thetif**
Fixes the first 3 issues in #2607

### This pull request changes...

- skip link should be visible and working
- changed the way primary key personnel were added so there will not be an empty 1. header
- change the arial labels for the previous/continue buttons so they specified Previous or Continue and then said where it was going

### This pull request is ready to merge when...

- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
~- [ ] The change has been documented~
~- [ ] Associated OpenAPI documentation has been updated~
- [ ] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Review by jeromeleecms — COMMENTED** [review]
1. The skip link popup appears, BUT on the "Agree and Continue Screen" it doesn't take me anywhere. Shouldn't this take me to the "Agree and Continue Button?"
2. The skip link popup appears, BUT on the "Log In Form" it takes me to a nebulous area. Where should I expect for it to take me?
3. Can't login to the rest of the app to check behavior, but will follow up with the rest of the review once I can log in.

**Comment by thetif** [comment]
tried to resolve these issues, can you try it again?

File: web/src/styles/scss/layout.scss [hunk 1]
```diff
@@ -42,16 +42,19 @@ footer {
 }
 
 .skip-nav {
-  background: white;
+  background: #112e51;
   height: 3rem;
   line-height: 3rem;
-  padding: 0 0.5rem;
+  padding: 1rem;
+  width: fit-content;
```
Comment (tbolt) [inline web/src/styles/scss/layout.scss:49]: I'm not sure if this would have an impact but IE doesn't support fit-content

**Review by tbolt — COMMENTED** [review]
These updates look good. The skip nav behavior is baffling as it is inconsistent across browsers / operating systems. 

For the Key Personnel, I believe the "Edit / Remove" needs to be associated with the person added. I can understand if that needs to be part of a larger design update though

**Review by rohitpaul — APPROVED** [review]
Tested in Safari and Chrome.
Skip link visible.
Key Personal - Not blank to start off.
Continue Button and Previous Button - text is called out.

File: web/src/containers/Login.js [hunk 2]
```diff
@@ -59,9 +59,10 @@ const Login = ({
   }
 
   return (
-    <main id="start-main-content">
+    <main>
       <UpgradeBrowser />
       <CardForm
+        id="start-main-content"
```
[the remark this answers is not in the document]
Comment (thetif) [reply web/src/containers/Login.js:65]: yeah that was my thinking, do you think it's better to wrap the whole page?

[the remark this answers is not in the document]
Comment (tbolt) [reply web/src/containers/Login.js:65]: From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main
>The HTML `<main>` element represents the dominant content of the <body> of a document. The main content area consists of content that is directly related to or expands upon the central topic of a document, or the central functionality of an application.
I think wrapping the whole page (except the navigation in this case) makes sense.

File: web/src/styles/scss/layout.scss (hunk 1 above)
Comment (thetif) [reply web/src/styles/scss/layout.scss:49]: it doesn't really work anyway

File: web/src/components/ContinuePreviousButtons.js [hunk 3]
```diff
@@ -29,15 +32,14 @@ const ContinuePreviousButtons = ({ continueLink, previousLink }) => {
...
-      {buildLabel(continueLink)}
-    </p>
+  const continueLabel = !continueText ? null : (
+    <p className="ds-u-text-align--right" id="continue-text">{continueText}</p>
   );
 
   const previousComponent = !previousLink ? null : (
     <Link
-      aria-labelledby={previousLabelId}
+      aria-label={`Previous to ${previousText}`}
```
[the remark this answers is not in the document]
Comment (thetif) [reply web/src/components/ContinuePreviousButtons.js]: @beparticular suggested relabeling the button to Back instead of Previous

**Review by jeromeleecms — APPROVED** [review]
The skip link thing is still a mystery to me, I do the same thing but sometimes it behaves differently.. I'm going to say this is good to go and see what our 508 team comes up with as well.

**Pull request merged**